components/Inputs/input.css

```diff
@@ -0,0 +1,62 @@
+@import "../../style_guide.css";
+
+body {
+  background-color: var(--bg-color);
+  color: var(--txt-color);
+  overflow-x: hidden;
+}
+
+label {
+  display: block;
+}
```

Comment:
Try giving label the class name and then style it because it may break other label elements defined by the user also.

Reply:
changed it. Please check

Reply:
ok 👍🏻